lib/Doctrine/KeyValueStore/Storage/MongoDbStorage.php

```diff
@@ -112,55 +79,63 @@ public function requiresCompositePrimaryKeys()
      */
     public function insert($storageName, $key, array $data)
     {
-        $this->initialize();
-
-        $value = [
-            'key'   => $key,
-            'value' => $data,
-        ];
-
-        $this->collection->insert($value);
+        $this->client
+            ->selectDatabase($this->database)
+            ->selectCollection($storageName)
```

Comment:
This is not related to the MongoDB implementation per-se, but I wanted to point out the severity of the BC break.

Previously, all data was stored in the same collection, irrespective of the `$storageName` argument. While that may have been a violation of what's intended with the library, this BC break should be documented so people are aware of it.

Reply:
where do you suggest to document the BC? start an UPGRADE.md or a note in the [release description](https://github.com/doctrine/KeyValueStore/releases) could be enough?

Reply:
An `UPGRADE` document would probably be the best place, along with a note about the BC break in the release notes.